packages/cohere/src/cohere-chat-language-model.ts

```diff
@@ -108,6 +121,14 @@ export class CohereChatLanguageModel implements LanguageModelV2 {
 
         // documents for RAG:
         ...(cohereDocuments.length > 0 && { documents: cohereDocuments }),
+
+        // reasoning
+        ...(cohereOptions.thinking && {
+          thinking: {
+            type: cohereOptions.thinking.type ?? 'enabled',
+            token_budget: cohereOptions.thinking.tokenBudget,
+          },
+        }),
```

Comment:
in ai sdk 6 it might be worth revisiting default, i.e. should we go with provider defaults or with what we think are recommended settings that cut down time to get started for users

Reply:
Shall we keep track of thoughts like this somewhere? Shall we add a `TODO AI SDK 6` comment?